src/org/labkey/test/util/TestDataGenerator.java

```diff
@@ -411,4 +413,22 @@ public static TestDataGenerator createDomain(String containerPath, DomainProps d
 
         return def.getTestDataGenerator(containerPath);
     }
+
+    static public CommandResponse deleteDomain(final String containerPath, final String schema, final String domainName)
+            throws IOException, CommandException
+    {
+        Connection connection = WebTestHelper.getRemoteApiConnection();
+        DropDomainCommand delCmd = new DropDomainCommand(schema, domainName);
+        return delCmd.execute(connection, containerPath);
+    }
```

Comment:
I'd use the same terminology as the `DropDomainCommand` for these parameters; `query` or `queryName` instead of `domainName`

Reply:
Changed to queryName.